sphinx/websupport/__init__.py

```diff
@@ -12,11 +12,16 @@
 import warnings
 
 from sphinx.deprecation import RemovedInSphinx20Warning
-from sphinxcontrib.websupport import WebSupport  # NOQA
-from sphinxcontrib.websupport import errors  # NOQA
-from sphinxcontrib.websupport.search import BaseSearch, SEARCH_ADAPTERS  # NOQA
-from sphinxcontrib.websupport.storage import StorageBackend  # NOQA
 
-warnings.warn('sphinx.websupport module is now provided as sphinxcontrib.webuspport. '
-              'sphinx.websupport will be removed in Sphinx-2.0.  Please use it instaed',
-              RemovedInSphinx20Warning)
+try:
+    from sphinxcontrib.websupport import WebSupport  # NOQA
+    from sphinxcontrib.websupport import errors  # NOQA
+    from sphinxcontrib.websupport.search import BaseSearch, SEARCH_ADAPTERS  # NOQA
+    from sphinxcontrib.websupport.storage import StorageBackend  # NOQA
+
+    warnings.warn('sphinx.websupport module is now provided as sphinxcontrib-webuspport. '
+                  'sphinx.websupport will be removed in Sphinx-2.0.  Please use it instaed',
```

Comment:
Ah, sorry, somehow didn't notice this PR had already been merged :/, not paying enough attention!